PatchNotes/Altibase_7.1/eng/Altibase_7_1_0_9_9_Patch_Notes.md

```diff
@@ -0,0 +1,260 @@
+-   **How to reproduce this bug**
+
+    -   **Reproduction conditions**
+
+    -   **Actual Results**
+
+    -   **Expected Results**
+-   **Workaround**
+-   **Changes**
+    -   Performance view
+    -   Property
+    -   Compile Option
+    -   Error Code
+
+### BUG-50997<a name=bug-50997></a> A results error can occur under certain conditions when using an aggregate function in a subquery that references an outer query.
+
+-   **module** : qp
+
+-   **Category** : Functional Error
+
+-   **Reproducibility** : Always
+
+- **Description** : Altibase fixed an issue where a results error occurred under specific conditions when using an aggregate function in a subquery that references an outer query.
+
+  This bug only occurred when all of the following conditions were met:
+
+  1.  An `OR` operator is used within the conditions of an `AND` operation.
+  2. The condition in the `OR` operator contains a subquery that references the outer query.
+  3. The subquery mentioned in 2. uses an aggregate function.
+
+- **How to reproduce this bug**
+
+  - **Reproduction conditions**
+
+    ```sql
+    drop table T1;
+    drop table T2; 
+    
+    create table T1
+    (
+     REV integer,
+     REVTYPE smallint,
+     id integer
+    );
+    
+    insert into T1 values(7,              7,           0);
+    insert into T1 values(6,              5,           2);
+    insert into T1 values(5,              6,           1);
+    
+    create table T2
+    (
+     REV integer ,
+     REVTYPE smallint,
+     id integer
+    );
+    
+    insert into T2 values(15,          3,      2);           
+    insert into T2 values(17,          1,      0);           
+    insert into T2 values(10,          3,      1);           
+    
+    select A.id 
+    from T1 A, T2 B
+    where 
+       A.id = B.id
+      and 
+    (
+        (B.REV = (select min(BB.REV) from T2 BB where BB.REV < 23 and B.id = BB.id )
+          and 
+         A.REV = (select (AA.REV) from T1 AA where AA.REV < 23 and A.id = AA.id)
+        )
+      or  (
+        A.REVTYPE = 2
+         and B.REVTYPE <= 2
+           )
+    );
+    ```
+
+  - **Actual Results**
+
+    ```
+    ID
+    --------------
+    1
+    1 row selected.
+    ```
+
+  - **Expected Results**
+
+    ```
+    ID
+    --------------
+    0
+    2
+    1
+    3 rows selected.
+    ```
+
+- **Workaround**
+
+- **Changes**
+
+  -   Performance view
+  -   Property
+  -   Compile Option
+  -   Error Code
+
+### BUG-51004<a name=bug-51004></a> A "Function sequence error" may occur if the type or pointer of the variable bound to altibase_stmt_bind_param changes.
+
+-   **module** : ux-cdbc
+
+-   **Category** : Functional Error
+
+-   **Reproducibility** : Always
+
+-   **Description** : Altibase fixed the issue to prevent a "Function sequence error" when the type or pointer of the variable bound to altibase_stmt_bind_param changes.
```

Comment:
```suggestion
-   **Description** : Altibase fixed the issue to prevent a "Function sequence error" when the type or pointer of the variable bound to altibase_stmt_bind_param() changes.
```